.github/workflows/zombienet_cumulus.yml

```diff
@@ -329,6 +329,7 @@ jobs:
       CUMULUS_IMAGE: "${{ needs.preflight.outputs.TEMP_IMAGES_BASE }}/test-parachain:${{ needs.preflight.outputs.DOCKER_IMAGES_VERSION }}"
       RUST_LOG: ${{ needs.preflight.outputs.RUST_LOG }}
       ZOMBIE_PROVIDER: ${{ needs.preflight.outputs.ZOMBIE_PROVIDER }}
+      RUN_IN_CI: ${{ needs.preflight.outputs.RUN_IN_CI }}
```

Comment:
Soo we expect that this is always 1 right? (which makes sense since its a CI file). We fetch it from here? https://github.com/paritytech/polkadot-sdk/blob/4b83d24f4bc96a7b17964be94b178dd7b8f873b5/.github/zombienet-env#L7

Reply:
Yes, it is fetched from `zombienet-env`.
I think that eventually it might be simplified by setting `RUN_IN_CI` internally within `zombienet-sdk` GH action and not per each test to be executed - I guess it is safe to assume that GH action is run within CI :D

Reply:
dq: how does zn-sdk account for `RUN_IN_CI` ?

Reply:
it checks this flag and configures eg. ports accordingly.
See here:
https://github.com/paritytech/zombienet-sdk/blob/c801351b3f0e532754beaed8a82e666238b7afe5/crates/orchestrator/src/spawner.rs#L230C1-L236C13

Reply:
I think we can just set this env var in the workflow file, since will always be `1` in this context and put this in the preflight force us to duplicate this line in each job :(

Reply:
Refactored to to set this flag within zombienet-sdk GH action.
However, left option to set it via preflight in case some does not want to use zombienet-sdk GH action in his test.